### PR TITLE
Change JPEG -> SVG

### PR DESCRIPTION
**Pull request by domel**
- Change JPEG (raster image) to SVG (vector image). 
- Remove `example6.jpg` that wasn't used in HTML

See https://github.com/w3c/rdf-semantics/issues/42

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-semantics/pull/43.html" title="Last updated on Sep 11, 2023, 7:42 PM UTC (b089fc1)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-semantics/43/295f6fa...b089fc1.html" title="Last updated on Sep 11, 2023, 7:42 PM UTC (b089fc1)">Diff</a>

### CODE REVIEW

**Review by gkellogg — COMMENTED** [review]
If the SVG files have editable references, it would be good to put comments in place pointing to them, in case anything needs to be changed in the future.

**Comment by domel** [comment]
> If the SVG files have editable references, it would be good to put comments in place pointing to them, in case anything needs to be changed in the future.

@gkellogg What do you mean by "editable references"? Do you want to add (XML) comments in SVG files?

**Comment by domel** [comment]
BTW, here is a [Preview](https://raw.githack.com/w3c/rdf-semantics/svg_pictures/spec/index.html).

**Comment by gkellogg** [comment]
If you look at https://github.com/w3c/rdf-semantics/issues/42#issuecomment-1712579511 you’ll see an inline XML-style comment “Source for this image may be found …” in the code snippet. That’s what I was referring to.

**Comment by domel** [comment]
@gkellogg Oh yes. I didn't add it on purpose. Experienced by other PRs and @pfps comments, I preferred not to change to much the HTML structure. If there is consent to this change, I will be happy to add `<figure>`, even in the last image, it is worth adding `<picture>`.

**Comment by gkellogg** [comment]
There are some accessibility requirements for figures, but I think they mostly describe more explicit RDF statements. `alt`, and `aria-describedby` attributes are probably appropriate, but they don't necessarily need to be wrapped in a figure.

My point was that, in the case of RDF Concepts, [figure 2](https://www.w3.org/TR/rdf12-concepts/#fig-quoted-triple) includes the following comment:

```
<!-- Source for this file is at https://docs.google.com/drawings/d/1I_QxbUgnQXumXzb8c0WNJHIQ-mtRs2S80dDG6i9aOD8 -->
```

This allows someone to go to that URL and re-edit the figure to be able to export an update SVG if it ever becomes necessary. This can be put anywhere near the image.

**Comment by domel** [comment]
But I did the pictures locally in my computer so I guess that it can only link to the github repo.

**Comment by gkellogg** [comment]
If you did the pictures locally using some format other than SVG, it might be worth adding the source for these drawings to the repository with some README that indicated any software used to process them. Of course, if you created them by hand, then they can probably be edited by hand in the future.

**Comment by domel** [comment]
Yes, I created them by hand. I used the well-known Inkscape.

**Comment by gkellogg** [comment]
Cool, I think putting something about how the images are authored in a README would be useful. Trying to edit them by hand would be beyond me.

**Comment by domel** [comment]
@gkellogg Done

**Review by gkellogg — APPROVED** [review]

File: spec/RDF12SemanticsDiagrams/example5.svg
Comment (TallTed) [inline spec/RDF12SemanticsDiagrams/example5.svg:1]: The connecting lines/arrows should not be composed of two parts; they should each be a single line with the `ex:hasChild` label atop, as in `example3.svg`. As these are, it looks like `ex:Alice` and `ex:Bob` each point to their neighboring `ex:hasChild`, and that the two `ex:hasChild` each point to the central (blank) ellipse.

**Review by TallTed — CHANGES_REQUESTED** [review]
Small tweaks for `example5.svg`. Otherwise, these look just right!

File: spec/RDF12SemanticsDiagrams/example5.svg
Comment (domel) [reply spec/RDF12SemanticsDiagrams/example5.svg:1]: done

**Review by TallTed — APPROVED** [review]

**Pull request merged**